v2.0/restore.md

```diff
@@ -139,36 +146,46 @@ You can include the following options as key-value pairs in the `kv_option_list`
 
 ### Restore a Single Table
 
-``` sql
-> RESTORE bank.customers FROM 'azure://acme-co-backup/table-customer-2017-03-27-full?AZURE_ACCOUNT_KEY=hash&AZURE_ACCOUNT_NAME=acme-co';
-```
+~~~ sql
+> RESTORE bank.customers FROM 'gs://acme-co-backup/database-bank-2017-03-27-weekly';
+~~~
 
 ### Restore Multiple Tables
 
-``` sql
-> RESTORE bank.customers, accounts FROM 'azure://acme-co-backup/tables-accounts-customers-2017-03-27-full?AZURE_ACCOUNT_KEY=hash&AZURE_ACCOUNT_NAME=acme-co';
-```
+~~~ sql
+> RESTORE bank.customers, accounts FROM 'gs://acme-co-backup/database-bank-2017-03-27-weekly';
```

Comment:
Don't you need to qualify the second table name like you've done the first?

`bank.customers, bank.accounts`